src/OpenColorIO/transforms/builtins/ProPhotoRGB.cpp

```diff
@@ -0,0 +1,263 @@
+					const double absIn = std::abs(in);
+					double out = 0.0;
+
+					if (absIn < breakEnc)
+					{
+						out = absIn / slope;
+					}
+					else
+					{
+						out = std::pow(absIn, gamma);
+					}
+
+					return float(std::copysign(out, in));
+				};
+
+			CreateHalfLut(ops, GenerateLutValues);
+		}
+
+	} // namespace ROMM_RGB_GAMMA_18
+
+
+	// ProPhoto RGB with sRGB gamma curve.
+	// This is a common variant used by Adobe and other applications.
+	// Uses the sRGB transfer function (gamma 2.4, offset 0.055) instead of
+	// the standard ROMM RGB gamma 1.8 curve.
+	namespace ROMM_RGB_SRGB_GAMMA
+	{
+
+		void GenerateLinearToEncodedOps(OpRcPtrVec& ops)
+		{
+			// sRGB gamma encoding: gamma=2.4, offset=0.055
+			// This uses the MONCURVE model which is efficient for sRGB-style curves.
+			const GammaOpData::Params rgbParams = { 2.4, 0.055 };
+			const GammaOpData::Params alphaParams = { 1.0, 0.0 };
+			auto gammaData = std::make_shared<GammaOpData>(GammaOpData::MONCURVE_FWD,
+				rgbParams, rgbParams, rgbParams, alphaParams);
+			CreateGammaOp(ops, gammaData, TRANSFORM_DIR_FORWARD);
+		}
+
+		void GenerateEncodedToLinearOps(OpRcPtrVec& ops)
+		{
+			// sRGB gamma decoding: gamma=2.4, offset=0.055
+			const GammaOpData::Params rgbParams = { 2.4, 0.055 };
+			const GammaOpData::Params alphaParams = { 1.0, 0.0 };
+			auto gammaData = std::make_shared<GammaOpData>(GammaOpData::MONCURVE_REV,
+				rgbParams, rgbParams, rgbParams, alphaParams);
+			CreateGammaOp(ops, gammaData, TRANSFORM_DIR_FORWARD);
+		}
+
+	} // namespace ROMM_RGB_SRGB_GAMMA
+
+
+	namespace PROPHOTO
+	{
+
+		void RegisterAll(BuiltinTransformRegistryImpl& registry) noexcept
+		{
+			// Linear ProPhoto RGB to ACES2065-1.
+			{
+				auto ROMM_RGB_to_ACES2065_1_Functor = [](OpRcPtrVec& ops)
+					{
+						// Convert from ROMM RGB (D50) to ACES AP0 (D60).
+						// Uses Bradford chromatic adaptation.
+						MatrixOpData::MatrixArrayPtr matrix
+							= build_conversion_matrix(ROMM_RGB::primaries,
+								ACES_AP0::primaries,
+								ADAPTATION_BRADFORD);
+						CreateMatrixOp(ops, matrix, TRANSFORM_DIR_FORWARD);
+					};
+
+				registry.addBuiltin("PROPHOTO-RGB_to_ACES2065-1",
+					"Convert ProPhoto RGB (linear) to ACES2065-1",
+					ROMM_RGB_to_ACES2065_1_Functor);
+			}
+
+			// Encoded ProPhoto RGB (gamma 1.8) to ACES2065-1.
+			{
+				auto ROMM_RGB_ENCODED_to_ACES2065_1_Functor = [](OpRcPtrVec& ops)
+					{
+						// 1. Decode gamma 1.8 to linear.
+						ROMM_RGB_GAMMA_18::GenerateEncodedToLinearOps(ops);
+
+						// 2. Convert color space from ROMM RGB (D50) to ACES AP0 (D60).
+						MatrixOpData::MatrixArrayPtr matrix
+							= build_conversion_matrix(ROMM_RGB::primaries,
+								ACES_AP0::primaries,
+								ADAPTATION_BRADFORD);
+						CreateMatrixOp(ops, matrix, TRANSFORM_DIR_FORWARD);
+					};
+
+				registry.addBuiltin("PROPHOTO-RGB-ENCODED_to_ACES2065-1",
+					"Convert ProPhoto RGB (gamma 1.8 encoded) to ACES2065-1",
+					ROMM_RGB_ENCODED_to_ACES2065_1_Functor);
```

Comment:
ROMM/ProPhoto is display-referred, so the built-in should go to the display-referred connection space, CIE-XYZ-D65.

I would propose naming this "ROMM_to_CIE-XYZ-D65_BFD".